Cargo.toml

```
@@ -16,3 +23,59 @@ members = [
 	"ethereum-types",
 	"ethbloom",
 ]
+
+[workspace.dependencies]
+serde_json = "1.0.41"
+criterion = "0.5.1"
+rand = { version = "0.8.0", default-features = false }
+hex-literal = "0.4.1"
+scale-info = { version = ">=1.0, <3", default-features = false }
+quickcheck = "1"
+rustc-hex = { version = "2.0.1", default-features = false }
+static_assertions = "1.0.0"
+arbitrary = "1.0"
+tiny-keccak = "2.0"
+crunchy = { version = "0.2.2", default-features = false }
+serde = { version = "1.0.101", default-features = false }
+codec = { package = "parity-scale-codec", version = "3.7.4", default-features = false }
+log = { version = "0.4.17", default-features = false }
+schemars = ">=0.8.12"
+rand_xorshift = "0.3.0"
+tempfile = "3.1.0"
+smallvec = "1.0.0"
+parking_lot = "0.12.0"
+num_cpus = "1.10.1"
+regex = "1.3.1"
+rocksdb = { version = "0.22.0", default-features = false }
+alloc_counter = "0.0.4"
+sysinfo = "0.30.13"
+ctrlc = "3.1.4"
+chrono = "0.4"
+num-traits = { version = "0.2", default-features = false }
+integer-sqrt = "0.1"
+bytes = { version = "1", default-features = false }
+syn = "2.0.72"
+quote = "1.0.2"
+proc-macro2 = "1.0.8"
+byteorder = { version = "1.4.2", default-features = false }
+hex = { version = "0.4", default-features = false }
+num-bigint = "0.4.0"
+rug = { version = "1.6.0", default-features = false }
+jsonschema = { version = "0.23", default-features = false }
+serde_derive = "1.0.101"
+
+ethbloom = { path = "./ethbloom", default-features = false }
+ethereum-types = { path = "./ethereum-types" }
+fixed-hash = { path = "./fixed-hash", default-features = false }
+uint = { path = "./uint", default-features = false }
+uint-crate = { path = "./uint", package = "uint", default-features = false }
+primitive-types = { path = "./primitive-types", default-features = false }
+impl-codec = { path = "./primitive-types/impls/codec", default-features = false }
+impl-num-traits = { path = "./primitive-types/impls/num-traits", default-features = false }
+impl-rlp = { path = "./primitive-types/impls/rlp", version = "0.4", default-features = false }
```

we don't seem to provide local version elsewhere
```suggestion
impl-rlp = { path = "./primitive-types/impls/rlp", default-features = false }
```